packages/twenty-front/src/modules/object-record/spreadsheet-import/utils/buildRecordFromImportedStructuredRow.ts

```diff
@@ -145,7 +202,7 @@ export const buildRecordFromImportedStructuredRow = ({
     },
   };
 
-  for (const field of fields) {
+  for (const field of fieldMetadataItems) {
     const importedFieldValue = importedStructuredRow[field.name];
 
     switch (field.type) {
```

Comment:
an assertUnreachable may help us as well here (for instance the DATE type we found that the empty string was probematic)